tests/molecules/C2_pp/CMakeLists.txt

```diff
@@ -1,54 +1,70 @@
-
+IF (ENABLE_SOA)
```

Comment:
I guess the AoS should also work. MSD operates at the level above SPOSet.

Reply:
It still works for the XML version. The point is to have the HDF5 version by default.  I will fix.